README.md

```diff
@@ -15,7 +15,21 @@
 
 ## Pyomo Overview
 
-Pyomo is a Python-based open-source software package that supports a diverse set of optimization capabilities for formulating and analyzing optimization models.
+Pyomo is a Python-based open-source software package that supports a diverse set of optimization capabilities for formulating and analyzing optimization models. Pyomo can be used to define symbolic problems, create concrete problem instances, and solve these instances with standard solvers. Pyomo supports a wide range of problem types, including:
+
+ -  Linear programming
+ -  Quadratic programming
+ -  Nonlinear programming
+ -  Mixed-integer linear programming
+ -  Mixed-integer quadratic programming
+ -  Mixed-integer nonlinear programming
+ -  Mixed-integer stochastic programming
+ -  Generalized disjunctive programming
+ -  Differential algebraic equations
+ -  Bilevel programming
```

Comment:
While we are at it, we should remove Bilevel, as that package is currently deprecated.